pkg/lns/chirpstack.go

```diff
@@ -154,7 +154,7 @@ func (c *ChirpstackClient) Listen(ch chan<- types.DeviceEvent) {
 							fmt.Printf("[lns][%s] device joined %s\n", msg.ID, pl.DevEui)
 							device, err := c.GetDevice(pl.DevEui)
 							if err != nil {
-								log.Printf("Error while joining device :: %s\n", err)
+								fmt.Printf("[lns][error][%s] error while joining device; cant get device %s\n", msg.ID, err)
```

Comment:
`error while joining device; cant get device %s :: %s\n`  <= should be `error fetching device from lns; skipping helium update for device join: %s :: %s\n`  (we only listen for events here; we received the device join event from Chirpstack, tried to fetch the device; but that failed)